javascript/401-code-challenges/graph/index.js

```diff
@@ -38,6 +37,34 @@ class Graph {
   size() {
     return this.graph.size;
   }
+
+  // code challenge 36, graph breadth first implementation
+  breadthFirst(node) {
+    if (!node) {
+      return [];
+    }
```

Comment:
What happens if you give it a node that is not in the graph? If you want better data validation, try
```js
if (!this.graph.has(node) {
  return []
}
```